package.json

```diff
@@ -55,6 +55,6 @@
     "underscore": "1.6.0"
   },
   "peerDependencies": {
-    "chai": ">= 1.6.1 < 2"
+    "chai": ">= 2"
```

Comment:
You might want to use the following instead: `"chai": ">= 1.6.1 < 3"`. Check sinon-chai [package.json](https://github.com/domenic/sinon-chai/blob/master/package.json#l36)